tests/test_database.py

```diff
@@ -38,41 +79,112 @@ def setUpClass(self):
             self.test_database_connector)
         self.test_tag_manager = database_manager.TagManager(
             self.test_database_connector)
+        self.test_search_manager = search_manager.SearchManager(
```

Comment:
You probably don't have to prefix stuff with `test_` since that should be quite clear to the reader this is all a test

Reply:
fixed